src/app/shared/date.util.ts

```diff
@@ -31,21 +31,7 @@ export function dateToISOFormat(date: Date | NgbDateStruct | string): string {
   const dateObj: Date = (date instanceof Date) ? date :
     ((typeof date === 'string') ? ngbDateStructToDate(stringToNgbDateStruct(date)) : ngbDateStructToDate(date));
```

Comment:
Now that this method is so much simpler this nested ternary thing stands out a lot...

Reply:
Agreed, @ybnd that this is a bit of ugly code. That said, I honestly couldn't find a way to easily pull this apart without requiring more refactors (which I didn't have time to work on).  So, while this is ugly, it appears to work as expected based on the new tests.  I'm planning to just leave this as-is for now, until someone has a chance/reason to clean this code up further.

Reply:
ok 👍